fiul-service-consent/src/main/java/io/finarkein/fiul/consent/impl/ConsentServiceImpl.java

```diff
@@ -109,23 +109,17 @@
         return buildFromConsentState(consentHandle)
                 .get()
                 .map(Mono::just)
```

Comment:
This still appears to be an imposter reactive action!

Reply:
fixes in this flow will be done in future checkin